CoreUtilities/src/au/gov/asd/tac/constellation/utilities/text/SpellCheckingTextArea.java

```diff
@@ -184,47 +179,155 @@ private BooleanBinding getSelectionBinding() {
             return selectionRange == null || selectionRange.getLength() == 0;
         }, this.selectionProperty());
     }
-
-    public void autoComplete(final List<String> suggestions) {
```

Comment:
Should probably add this back?

Reply:
Verified after removing this block auto complete still works as expected for selected item from suggestions list.